docs/ml/deploy.md

```diff
@@ -50,6 +50,11 @@ For some models of the ML model service, like the [Triton ML model service](http
 
 ## Create an ML model service
 
+The Viam registry hosts trained ML models that users have made public, which you can use to deploy classifiers or detectors for your use case onto your robot instead of training your own.
```

Comment:
```suggestion
The Viam registry hosts trained ML models that users have made public, which you can use to deploy classifiers or detectors for your use case onto your machine instead of training your own.
```